### PR TITLE
Expose optional granular API to help users minimise parseHTML (closes #5)

### LINKED ISSUE #5: Provide a way to clone parsed element to avoid parseHTML
screen shot 2015-08-26 at 20 00 12

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@etiennesegonzac r?

**Comment by wilsonpage** [comment]
This seemed like the simplest way to give this type of optional control to the user and not spoil nice API :)

I thought about adding a `poplar.clone(el)` API, but that would require that we always keep a reference to the original formatted element which is a waste of memory most of the time. This approach empowers the user to make the right decision taking the burden off us :)

File: test/test.js [hunk 1]
```diff
@@ -51,4 +51,20 @@ suite('poplar >', function() {
...
+    var el3 = poplar.create(parsed.cloneNode(true));
+
+    poplar.populate(el1, { name: '1' });
+    poplar.populate(el2, { name: '2' });
+    poplar.populate(el3, { name: '3' });
+
+    assert.equal(el1.textContent, '1');
+    assert.equal(el2.textContent, '2');
+    assert.equal(el3.textContent, '3');
+  });
```
Comment (etiennesegonzac) [inline test/test.js:69]: :+1:

**Comment by etiennesegonzac** [comment]
ship it!

**Pull request merged**